xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -489,6 +489,19 @@ $displayName##
  *#
 #macro(livetablecallback $divid $collist $colprops $transprefix)
 function (row, i, table) {
+   ## Make sure the columns containing HTML are formatted in HTML indeed
+   #set ($htmlColumnKeys = ['doc.location'])
+   #foreach($htmlColumnKey in $htmlColumnKeys)
+     #if ($collist.contains($htmlColumnKey))
+       #if (!$colprops[$htmlColumnKey])
+         #set ($colprops[$htmlColumnKey] = {})
+       #end
+       #if ($colprops[$htmlColumnKey].html == $NULL)
+         #set ($colprops[$htmlColumnKey].html = true)
```

Comment:
I would still put ``$colprops[$htmlColumnKey]`` in a separate variable as you use it 4 times.